_posts/2017/2017-06-13-reactive-streams.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: "Reactive streams"
+date: "2017-06-13 08:00:00"
+image: blog-header/api-journey.jpg
+categories: tech-stories
+authors: ["Christian"]
+---
+
+# akka-streams
+
+Reactive is one of the buzzwords of our time.
+Since one thread per request is so 90s and so all our beloved microservices have to be reactive, that is responsive, elastic, resilient and message-driven - whatever that means.
+Moreover we understand that modern applications need to deliver information in realtime.
+Batch processing is so 80s and so all our beloved microservices have to process data as it gets in and forward the new intermediate results to other microservices as soon as possible which in turn also process the data the moment it arrives.
```

Comment:
Like "have to process data as soon as it arrives". So basically immediately, yes